src/styles/movies.scss

```diff
@@ -0,0 +1,300 @@
+.container {
```

Comment:
Consider moving component styled to the corresponding component's directory for a better project structure, making it easier to find at the same time.